### DISCUSSION TITLE
Issue checking out lisf-public-7.x branch with git

### DISCUSSION THREAD

**Discussion by jessiemoravek** [NASA-LIS/LISF#1440] [Q&A] [answered]
Hello, 

I am working through the LISF Quick Start Guide. I followed the steps in the "working with github" guide and am now trying to use [git checkout:](https://github.com/NASA-LIS/LISF/blob/master/docs/discover_quick_start/discover_quick_start.adoc#compiling-lisf-components) 

`git checkout support/lisf-public-7.4`

However, I'm getting the following error: 
`error: pathspec 'support/lisf-public-7.4' did not match any file(s) known to git`

When I run git remote -v, I get the following: 
```
origin  github:jessiemoravek/LISF.git (fetch)
origin  github:jessiemoravek/LISF.git (push)
upstream        github:NASA-LIS/LISF.git (fetch)
upstream        github:NASA-LIS/LISF.git (push)

```

This seems correct to me, so I don't understand why the checkout isn't working. Any suggestions would be appreciated!

Thank you, 
Jessie

**Comment by jvgeiger** [comment, marked as answer]
Hello,

Thanks for reading the docs.  :-)

I am sorry that this note did not stand out enough:

If this is your first time working with upstream/<support/branch> (or you removed that tracking branch), then

```
% git fetch upstream <support/branch>
% git checkout -b <support/branch> upstream/<support/branch>
```

So for your case, run

```
% git fetch upstream support/lisf-public-7.4
% git checkout -b support/lisf-public-7.4 upstream/support/lisf-public-7.4
```

That will checkout the lisf-public-7.4 branch.
